s3-sink-connector/src/integration-test/java/io/aiven/kafka/connect/AvroIntegrationTest.java

```diff
@@ -281,6 +281,7 @@ private KafkaProducer<String, GenericRecord> newProducer() {
         producerProps.put(ProducerConfig.VALUE_SERIALIZER_CLASS_CONFIG,
                 "io.confluent.kafka.serializers.KafkaAvroSerializer");
         producerProps.put("schema.registry.url", SCHEMA_REGISTRY.getSchemaRegistryUrl());
+        producerProps.put("linger.ms", 1000);
```

Comment:
linger.ms was added to send all the test events in one batch, so that the flush method is not called in between small batches of kafka events being sent causing the integration tests to fail.